CHANGELOG.md

```diff
@@ -1,120 +1,157 @@
 # 1.3.0
-
-* Updated frost-link to accept named parameters in addition to position parameters, which unblocks the `spread` use case
-
+
+* Updated frost-link to accept named parameters in addition to position parameters, which unblocks the `spread` use case
+
```

Comment:
@dafortin Are you using Windows by any chance? Or an editor that uses carriage-returns? I'm noticing what appear to be line ending changes here.